### PR TITLE
Features/refactor return types fix tests

### PR DESCRIPTION
**Pull request by lxlang**
fix return types and updated namespacing

### CODE REVIEW

**Comment by lxlang** [comment]
@chmoelders please review

File: composer.json [hunk 1]
```diff
@@ -15,12 +15,13 @@
...
 	},
-	"suggests": {
+	"suggest": {
 		"ext-curl": "*"
 	},
 	"autoload": {
-		"psr-4": {
-			"Sofort\\SofortLib\\": "src"
-		}
+		"psr-0": {"Sofort\\SofortLib\\": "src"}
```
Comment (0x20h) [inline composer.json:22]: It's PSR-4, isn't it?

Comment (lxlang) [inline composer.json:22]: I moved this to a psr-0. So I can autoload the test-sources by psr-4 without having any troubles in the autoloader.

**Comment by chmoelders** [comment]
Nice job. Would you like to clean up my kitchen as well?

**Pull request merged**